.travis.yml

```diff
@@ -2,7 +2,7 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode7.3
```

Comment:
So they make mention of building without C++11 support as they don't have any. However, the copy of `clang++` in XCode 6.1 does have C++11 support. Did you try adding `-std=c++11` to the C++ flags? I don't know if it would help because maybe it is autodetected and added anyways, but it might be worth trying.